src/main/java/seedu/address/logic/commands/SellCommand.java

```diff
@@ -71,6 +73,7 @@ public CommandResult execute(Model model) throws CommandException {
         Company editedCompany = companyToEdit;
         editedCompany.addTransaction(transaction);
         model.setCompany(companyToEdit, editedCompany);
+        model.updateFilteredCompanyList(PREDICATE_SHOW_ALL_COMPANIES);
```

Comment:
Perhaps this line should be deleted in accordance with #151.

Reply:
Okay will do thank you :)